2019-governance-committee-election.md

```diff
@@ -0,0 +1,141 @@
+# OpenTelemetry 2019 Governance Committee election
+
+This document lays out the election schedule and process for the OpenTelemetry
+governance committee to implement the [charter
+document](https://github.com/open-telemetry/community/blob/master/governance-charter.md#establishment-of-a-governance-committee)
```

Comment:
nit: use `[charter][charter]` and add `[charter]: <url>` at the bottom of the document, since you're referring to it many times, and the long URL makes the document hard to read